x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents.tsx

```diff
@@ -6,23 +6,45 @@
 
 import React from 'react';
 
-import {
-  EuiPageHeader,
-  EuiPageHeaderSection,
-  EuiTitle,
-  EuiPageContent,
-  EuiPageContentBody,
-} from '@elastic/eui';
+import { EuiPageHeader, EuiPageHeaderSection, EuiTitle, EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { useValues } from 'kea';
+import { i18n } from '@kbn/i18n';
 
+import { DocumentCreationButton } from './document_creation_button';
 import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 import { FlashMessages } from '../../../shared/flash_messages';
 import { DOCUMENTS_TITLE } from './constants';
+import { EngineLogic } from '../engine';
+import { AppLogic } from '../../app_logic';
+import { SearchExperience } from './search_experience';
+
+const MetaEngineCallout: React.FC = () => (
```

Comment:
Not a strong opinion, just a suggestion - I don't see a super strong advantage to pulling this out as a separate const/component in the same file vs just keeping it inline since we don't need to reuse it for DRY purposes. I'd say either keep it inline or split it out totally to a separate component file, but that's pretty minor / not a blocker

Reply:
Yeah, this is how it was originally so I left it in tact. It's actually kind of a pain to test like this as well, as shallow doesn't render the MetaEngineCallout, so you need to dive into it to do any tests related to it.

Reply:
Yeah for sure, I'd be a +1 to changing instances like these to inline for easier testing

Reply:
8d2b9ae59b6ed1c339f3887c76efd180651351a5